packages/spotlib/spotlib.2.5.3/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+authors: [ "jun.furuse@gmail.com" ]
```

Comment:
On Tue, 23 Jun 2015 23:39:00 -0700, camlspotter wrote:
> 
> In packages/spotlib/spotlib.2.5.3/opam:
> 
> > @@ -0,0 +1,22 @@
> > +opam-version: "1.2"
> > +authors: [ "jun.furuse@gmail.com" ]
> 
> Thanks @Chris00. I use omake therefore not a big fan of Oasis and am
> not really motivated to provide a good _oasis. Anyway, let me check
> my build scripts to see it is easy to transfer some information of
> opam and descr to _oasis.

A pluging for oasis to support omake would certainly be good.  Maybe
you want to open an issue for this?

Also, it would be nice if you could add a `findlib` file (to this PR)
listing the provided (findlib) libraries — one per line.